lib/src/platform_client.dart

```diff
@@ -584,6 +584,21 @@ class PlatformClient {
     );
   }
 
+  /// Used to update the preferred ai language level of a user.
+  Future<void> updateAiLanguageLevel(AiLanguageLevel languageLevel) async {
+    await setUserProperty(
```

Comment:
Not a big deal but I would recommend returning the future itself instead of waiting for it inside the function. Since you're not doing anything after the await. 

```dart
Future<void> updateAiLanguageLevel(AiLanguageLevel languageLevel) {
    return setUserProperty(
```

Reply:
Actually, i didn't pay enough attention to this matter as i just copied existing similar functions and go : )). but i agree 100%